modules/openapi-generator/src/main/resources/go/go.mod.mustache

```diff
@@ -0,0 +1,6 @@
+module github.com/{{gitUserId}}/{{gitRepoId}}
```

Comment:
This all seems to be very specific to modules in GitHub repos. Personally I would think about making the complete module path as a input variable to the generator...

Reply:
Yup, we talked about this as well. No one has asked for supporting other git hosting services so far (which is a bit surprising to me). We can introduce another option "gitHost" if there's a need to support Gitlab, BitBucket or other Git hosting services.

Reply:
Okay, fair enough. For me, it feels a bit clunky to have to specify these two (and perhaps in future three)  pieces of information separately on the command line.

Reply:
Yup, agreed some users prefer just one single option for this particular setting. gitUserID and gitRepoID are global options used by other generators as well (e.g. PHP) so another benefit is consistency among generators. Let's go with the current approach and see if more Go users have an opinion on this.